pkg/cli/log_test.go

```diff
@@ -0,0 +1,127 @@
+package cli
+
+import (
+	"github.com/acorn-io/acorn/pkg/cli/testdata"
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestLog(t *testing.T) {
+	type fields struct {
+		Quiet  bool
+		Output string
+		All    bool
+	}
+	type args struct {
+		cmd    *cobra.Command
+		args   []string
+		client *testdata.MockClient
+	}
+	var _, w, _ = os.Pipe()
+	tests := []struct {
+		name           string
+		fields         fields
+		args           args
+		wantErr        bool
+		wantOut        string
+		commandContext client.CommandContext
+	}{
+		{
+			name: "acorn logs found", fields: fields{
+				All:    false,
+				Quiet:  false,
+				Output: "",
+			},
+			commandContext: client.CommandContext{
+				ClientFactory: &testdata.MockClientFactory{},
+				StdOut:        w,
+				StdErr:        w,
+				StdIn:         strings.NewReader(""),
+			},
+			args: args{
+				args:   []string{"found"},
+				client: &testdata.MockClient{},
+			},
+			wantErr: false,
+			wantOut: "",
```

Comment:
This looks like the success state, should we not have some output here?

Reply:
there are not any logs being outputted by that container so no output